code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -173,6 +173,12 @@
 		new /obj/item/stack/sheet/wetleather(get_turf(HH), HH.amount)
 		qdel(HH)
 
+/datum/reagent/water/on_mob_life(mob/living/carbon/M)
+	for(var/datum/reagent/R in M.reagents.reagent_list)
+		if(R != src)
+			M.reagents.remove_reagent(R.type,0.25)
```

Comment:
I feel like this should be smaller, like 0.1 since it should be insanely weak since its just... well you know, water